tests/admin/controller/test_analytics_services.py

```diff
@@ -204,24 +211,18 @@ def test_analytics_services_post_create(self):
             response = self.manager.admin_analytics_services_controller.process_analytics_services()
             eq_(response.status_code, 201)
 
-        service = get_one(self._db, ExternalIntegration, goal=ExternalIntegration.ANALYTICS_GOAL)
+        service = get_one(
+            self._db, ExternalIntegration,
+            goal=ExternalIntegration.ANALYTICS_GOAL,
+            protocol=GoogleAnalyticsProvider.__module__
+        )
         eq_(service.id, int(response.response[0]))
         eq_(GoogleAnalyticsProvider.__module__, service.protocol)
         eq_("http://test", service.url)
         eq_([library], service.libraries)
         eq_("trackingid", ConfigurationSetting.for_library_and_externalintegration(
                 self._db, GoogleAnalyticsProvider.TRACKING_ID, library, service).value)
 
-        # Creating a local analytics service doesn't require a URL.
```

Comment:
Since we already have these tests, it seems useful to keep them in (by deleting the local analytics service beforehand). That way if we ever go back to not _always_ having an analytics service, we'll still have the code coverage.